packages/catalog-process/src/utilities/config.ts

```diff
@@ -1,47 +1,60 @@
 import * as dotenvFlow from "dotenv-flow";
 import { z } from "zod";
-import { config as commonsConfig } from "pagopa-interop-commons";
+import { config as commonsConfig, CommonConfig } from "pagopa-interop-commons";
```

Comment:
we could rename to `Config` in commons and use `Config as CommonsConfig` for uniformity with the import `config as commonsConfig`